### PR TITLE
feat(condo): DOMA-8338 new import modals

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
1. Added [new import modals](https://www.figma.com/file/NnpWfYUo4Z8LGV3gHXRnHe/%D0%9E%D0%BD%D0%B1%D0%BE%D1%80%D0%B4%D0%B8%D0%BD%D0%B3?type=design&node-id=2245-15303&mode=design&t=rKXKwUvcIHZT06ob-0) on contact/meter/ticket/property pages
2. Remove unused code from `apps/condo/domains/common/components/containers`

изображение
изображение
изображение

### CODE REVIEW

File: apps/condo/lang/ru/ru.json [hunk 1]
```diff
@@ -24,15 +24,38 @@
...
-  "containers.FormTableExcelImport.SelectColumn": "Выбери колонку",
-  "containers.FormTableExcelImport.Step1Text": "1. Загрузите файл",
-  "containers.FormTableExcelImport.Step2Text": "2. Укажите, каким колонкам соответствуют данные и нажмите \"продолжить\"",
-  "containers.FormTableExcelImport.Step3Text": "3. Отредактируйте и сохраните данные",
+  "import.uploadButtonLabel": "Импорт списка из файла",
+  "import.uploadModal.title": "Добавьте {plural} на платформу",
+  "import.uploadModal.chooseFileForUpload": "Выбрать файл для загрузки",
+  "import.uploadModal.message": "Загрузите Excel-файл с информацией о {prepositional} или воспользуйтесь шаблоном",
+  "import.uploadModal.requiredFields.title": "Заполните обязательные поля",
+  "import.uploadModal.exampleLinkMessage": "Шаблон списка {genitive}.xsl",
```
Comment (Alllex202) [inline apps/condo/lang/ru/ru.json:32]:
```suggestion
  "import.uploadModal.exampleLinkMessage": "Шаблон списка {genitive}.xlsx",
```

File: apps/condo/lang/en/en.json [hunk 2]
```diff
@@ -24,15 +24,38 @@
...
-  "containers.FormTableExcelImport.SelectColumn": "Select column",
-  "containers.FormTableExcelImport.Step1Text": "1. Upload file",
-  "containers.FormTableExcelImport.Step2Text": "2. Choose, which columns match the data and click \"next\"",
-  "containers.FormTableExcelImport.Step3Text": "3. Edit and save data",
+  "import.uploadButtonLabel": "Import list from file",
+  "import.uploadModal.title": "Add {plural} to the platform",
+  "import.uploadModal.chooseFileForUpload": "Select file to download",
+  "import.uploadModal.message": "Download the Excel file with {prepositional} information or use the template",
+  "import.uploadModal.requiredFields.title": "Fill in the required fields",
+  "import.uploadModal.exampleLinkMessage": "{genitive} list template.xsl",
```
Comment (Alllex202) [inline apps/condo/lang/en/en.json:32]:
```suggestion
  "import.uploadModal.exampleLinkMessage": "{genitive} list template.xlsx",
```

File: apps/condo/pages/contact/index.tsx [hunk 3]
```diff
@@ -121,16 +118,16 @@ export const ContactsPageContent = ({
                         accessCheck={canManageContacts}
                         button={(
                             <ImportWrapper
-                                objectsName={ContactsMessage}
                                 accessCheck={canManageContacts}
                                 onFinish={refetch}
                                 columns={columns}
                                 maxTableLength={maxTableLength}
```
Comment (Alllex202) [inline apps/condo/pages/contact/index.tsx:124]: Can we move `maxTableLength` into a common component?

File: apps/condo/public/ticket-import-example.svg
Comment (Alllex202) [inline apps/condo/public/ticket-import-example.svg:1]: Where are examples for contacts and meters?

File: apps/condo/public/property-import-example.svg
Comment (Alllex202) [inline apps/condo/public/property-import-example.svg:1]: These examples are in Russian only. What about other languages?

Comment (Alllex202) [reply apps/condo/public/property-import-example.svg:1]: You should not show examples in Russian if another locale is specified (not RU).
Or you should make examples in different languages.

File: apps/condo/public/ticket-import-example.svg
Comment (nomerdvadcatpyat) [reply apps/condo/public/ticket-import-example.svg:1]: added

File: apps/condo/domains/common/components/Import/Index.tsx [hunk 4]
```diff
@@ -142,80 +155,217 @@ const ImportWrapper: React.FC<IImportWrapperProps> = (props) => {
...
+    const { useFlagValue } = useFeatureFlags()
+    const defaultMaxTableLength: number = useFlagValue(BIGGER_LIMIT_FOR_IMPORT) || DEFAULT_RECORDS_LIMIT_FOR_IMPORT
+
     const [importData, progress, error, isImported, breakImport] = useImporter({
         columns,
         rowNormalizer,
         rowValidator,
         objectCreator,
-        maxTableLength,
+        maxTableLength: maxTableLength || defaultMaxTableLength,
```
Comment (Alllex202) [inline apps/condo/domains/common/components/Import/Index.tsx:166]: What is this logic for? Now we use the same limits everywhere

File: apps/condo/domains/property/components/BuildingsTable.tsx [hunk 5]
```diff
@@ -182,16 +162,13 @@ export default function BuildingsTable (props: BuildingTableProps) {
...
-                        maxTableLength={maxTableLength}
                         rowNormalizer={propertyNormalizer}
                         rowValidator={propertyValidator}
-                        domainTranslate={PropertyTitle}
                         objectCreator={propertyCreator}
-                        exampleTemplateLink={exampleTemplateLink}
+                        domainName='property'
                     >
                         <Button
                             type='secondary'
```
Comment (Alllex202) [inline apps/condo/domains/property/components/BuildingsTable.tsx:174]: Right now child elements are not used anywhere in this component

Comment (Alllex202) [reply apps/condo/domains/property/components/BuildingsTable.tsx:174]: And on other pages too

Comment (Alllex202) [reply apps/condo/domains/property/components/BuildingsTable.tsx:174]: And check that it looks good in all cases:
- table is empty
- table with rows

File: apps/condo/domains/common/components/Import/Index.tsx [hunk 6]
```diff
@@ -142,80 +155,217 @@ const ImportWrapper: React.FC<IImportWrapperProps> = (props) => {
...
+                        </DataImporter>
+                    }
+                >
+                    <Space direction='vertical' size={24}>
+                        <Typography.Text>
+                            {UploadModalMessage}
+                        </Typography.Text>
+                        <ImageContaier>
+                            <img alt='example-import-image' src={exampleImageSrc} />
+                        </ImageContaier>
```
Comment (Alllex202) [inline apps/condo/domains/common/components/Import/Index.tsx:263]: Does it look good if the picture was not found in the specified path? For example, with `EN` locale

Comment (nomerdvadcatpyat) [reply apps/condo/domains/common/components/Import/Index.tsx:263]: Here will be a picture

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**